src/features/expense/ui/UnReviewedExpenseList/ClassifyExpenseDrawer.tsx

```
@@ -0,0 +1,182 @@
+interface RadioItemProps {
+  isSelected: boolean;
+  consumption: Consumption;
+  onClick: () => void;
+}
+
+const RadioItem: React.FC<RadioItemProps> = ({
+  isSelected,
+  consumption,
+  onClick,
+}) => {
+  const { title, description, info, consumptionKind } = consumption;
```

ESLint

[error] 'consumptionKind' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)